components/transactions/TransactionDetails.js

```diff
@@ -67,28 +67,30 @@ const TransactionDetails = ({
 
   return (
     <DetailsContainer flexWrap="wrap" alignItems="flex-start">
-      <Flex flexDirection="column" width={[1, 0.4]}>
-        {toAccount.host && (
-          <Box>
-            <DetailTitle>
-              <FormattedMessage id="Member.Role.FISCAL_HOST" defaultMessage="Fiscal Host" />
-            </DetailTitle>
-            <DetailDescription>
-              <LinkCollective collective={toAccount.host} />
-            </DetailDescription>
-          </Box>
-        )}
-        {order.paymentMethod && (
-          <Box>
-            <DetailTitle>
-              <FormattedMessage id="PaidWith" defaultMessage="Paid With" />
-            </DetailTitle>
-            <DetailDescription>
-              <PaymentMethodTypeWithIcon type={order.paymentMethod.type} fontSize={11} iconSize={16} />
-            </DetailDescription>
-          </Box>
-        )}
-      </Flex>
+      {toAccount.host && order?.paymentMethod && (
```

Comment:
I'm wondering why we should skip this if no `paymentMethod`.

Reply:
Good catch, this should be an `||`.